src/snowflake/cli/plugins/snowpark/package/manager.py

```diff
@@ -83,7 +39,7 @@ def create_packages_zip(zip_name: str) -> str:
     return file_name
 
 
-def cleanup_after_install(func):
+def cleanup_packages_dir(func):
```

Comment:
In my opinion we should get rid of this wrapper. But we can do this in follow-up PR